tensorflow/contrib/makefile/Makefile

```diff
@@ -451,6 +452,7 @@ $(wildcard tensorflow/core/common_runtime/gpu_device_factory.*)
 TF_CC_SRCS := $(filter-out $(CORE_CC_EXCLUDE_SRCS), $(CORE_CC_ALL_SRCS))
 # Add in any extra files that don't fit the patterns easily
 TF_CC_SRCS += tensorflow/core/common_runtime/gpu/gpu_tracer.cc
+TF_CC_SRCS += tensorflow/core/kernels/deep_conv2d.cc
```

Comment:
Could you add this file to tensorflow/contrib/makefile/tf_op_files.txt instead? That is where the kernel files are typically added:
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/makefile#fixing-makefile-issues

Thanks!